kernel/src/hil/text_screen.rs

```diff
@@ -0,0 +1,96 @@
+//! Interface for text screen and displays.
+use crate::returncode::ReturnCode;
+
+pub trait TextScreen {
+    fn set_client(&self, client: Option<&'static dyn TextScreenClient>);
```

Comment:
```suggestion
pub trait TextScreen<'a> {
    fn set_client(&self, client: Option<&'a dyn TextScreenClient>);
```

I didn't notice this before, but browsing https://github.com/tock/tock/issues/1074 and I remembered that we want to avoid using `&'static` for clients.

Reply:
Done